src/utils.jl

```diff
@@ -6,9 +6,7 @@ const HEIGHT = Ref{Int}()
 const MODE = Ref{Symbol}(:default)
 
 function terminal_size(io)
-    ws = IOCTL.ioctl(io, IOCTL.TIOCGWINSZ)
-    # width, height
-    return (Int(ws.ws_col), Int(ws.ws_row))
+    return displaysize(stdout)
```

Comment:
I believe `displaysize` returns `lines` and `columns`, which is `height` and `width` but the code expects returning `width` and `height` in that order.

Reply:
You are also correct, it does reverse it.

Reply:
I have updated the fix, please let me know what you think, thank you!